atomic_reactor/plugins/pre_hide_files.py

```diff
@@ -43,8 +43,28 @@ def run(self):
         self._populate_start_file_lines()
         self._populate_end_file_lines()
 
-        for parent_image_id in self.workflow.builder.parent_images.values():
-            self._update_dockerfile(parent_image_id)
+        # Find limits of each Dockerfile stage
```

Comment:
Could be this whole block extracted into method like `_update_build_stages` to have `run` written as steps on the same level of abstraction

Reply:
Good idea. Pushed a change, see what you think.